scripts/chromium-revision

```diff
@@ -105,8 +106,9 @@ async function printVersionRevisions(version) {
     async function(state) {
       let platform = state.platforms[state.i - 1];
       if (!platform) return state.value;
+      let rev = state.range[1];
 
-      for (let rev = state.range[1]; rev >= state.range[0]; rev--) {
+      for (; rev >= state.range[0]; rev--) {
```

Comment:
I got an error with this variable using the latest LTS node with this script. Apparently, `let` hoisting has changed under the hood. The change makes sense though, as we're referencing `rev` outside of the loop down below.